File: app/src/main/java/com/cartenz/cartenztaxcore/feature/forgot/ForgotContract.java
package com.cartenz.cartenztaxcore.feature.forgot;


import com.cartenz.core.BasePresenter;
import com.cartenz.core.BaseView;

public interface ForgotContract {
    interface View extends BaseView<PresenterInterface> {
        void forgotResult(String error);

    }

    interface PresenterInterface extends BasePresenter<View> {
        void callForgot(String email);
    }
}

File: kotlinapp/src/main/java/com/cartenz/kotlinapp/feature/forgot/ForgotContract.kt
package com.cartenz.kotlinapp.feature.forgot

import com.cartenz.kotlin_core.CartenzBasePresenter
import com.cartenz.kotlin_core.CartenzBaseView

interface ForgotContract {
    interface ViewCartenz : CartenzBaseView<PresenterInterfaceCartenz> {
        fun forgotResult(error: String)

    }

    interface PresenterInterfaceCartenz : CartenzBasePresenter<ViewCartenz> {
        fun callForgot(email: String)
    }
}
